tsconfig.json

```diff
@@ -13,6 +13,7 @@
       "importHelpers": true
   },
   "include": [
-      "./**/*.ts"
+      "index.ts",
```

Comment:
This might change how things are nested in the `dist` folder. It's worth trashing the dist folder locally, running the build once on an old branch, looking at the output in `dist`, trashing and switching back and building again.

Reply:
Looked into it. There is no `dist` folder configured here. The built results live along the sources and are consumed from there. 🙀